modules/dnn/src/webgpu/include/tensor.hpp

```diff
@@ -12,13 +12,10 @@ class Buffer;
 class Tensor{
 public:
     Tensor(Format fmt = wFormatFp32);
-    Tensor( const void* data, size_t size_in_byte, 
-            wgpu::BufferUsage usage, 
-            Format fmt = wFormatInt32);  //uniform buffer
-    Tensor( const void* data, std::vector<int>& shape, 
-            wgpu::BufferUsage usage, 
+    Tensor( const void* data, size_t size_in_byte);  //uniform buffer
```

Comment:
Missing argument "Format fmt" ?

Reply:
It seems that format is not needed in uniform buffer.